salt/minion.py

```diff
@@ -1620,12 +1633,31 @@ def _send_req_async(self, load, timeout):
             )
             load["sig"] = sig
 
+#        channel = salt.channel.client.AsyncPubChannel.factory(self.opts)
+#
+#        try:
+#            log.warn('SENDING OLD SCHOOL ASYNC %r', load)
+#            payload = channel.send(load, tries=tries, timeout=timeout)
+#        except SaltReqTimeoutError as e:
+#            raise SaltClientError(
+#                "Attempt to authenticate with the salt master failed with timeout error"
+#            )
+#        finally:
+#            channel.close()
+
         with salt.utils.event.get_event(
             "minion", opts=self.opts, listen=False
         ) as event:
             ret = yield event.fire_event_async(
                 load, "__master_req_channel_payload", timeout=timeout
             )
+
+        with salt.transport.client.AsyncReqChannel.factory(self.opts) as channel:
+            log.warning('Sending this load %r', load)
+            ret = yield channel.send(
+                load, timeout=timeout, tries=self.opts["return_retry_tries"]
+            )
+
```

Comment:
For some reason, *this* is what appears to make #62933 work. But it's not great, because the syndic is going to have to create a new connection every time a req is sent to the MoM? :thinking:

Reply:
Ahhhh... never mind I guess that should actually be the *local* Syndic Master.

Reply:
:+1: Done